app/controllers/steps_controller.rb

```diff
@@ -14,6 +14,19 @@ def destroy_all
     head :no_content
   end
 
+  def skip_all
+    query = WorkflowStep.where(druid: params[:druid],
+                               active_version: true,
+                               workflow: params[:workflow])
+    parser = ProcessParser.new(process_from_request_body)
+    WorkflowStep.transaction do
+      query.each do |record|
+        record.update(status: 'skipped', note: parser.to_h[:note])
```

Comment:
I tried update_all originally, I also tried update_all to update the time and it caused a rubocop error, so I changed it to this.

Reply:
no strong preference from me, i probably would have done an `update_all` too, though i don't know the details of what works and doesn't ... also this way seems fine to me .. i think regardless its important to be in a transaction and set that updated_at column, which this will do

Reply:
ah, ok, well in that case this wfm as-is, thanks for the explanation!